src/class/emitter.helper.ts

```diff
@@ -0,0 +1,25 @@
+import * as forge from 'node-forge';
```

Comment:
prefer named import to avoid importing useless code. It is a good practice for treeshaking, especially when you distribute a package.

https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad